src/auth/mock/user.mock.ts

```diff
@@ -20,4 +20,4 @@ export const userMock: UserInterface = {
     userId: 'Something Seomthing',
   }
```

Comment:
What are you mocking and why?

Reply:
Mocking the endpoint as there is none there yet